azext_iot/monitor/parsers/strings.py

```diff
@@ -83,11 +83,8 @@ def duplicate_property_name(duplicate_prop_name, interfaces: list):
 
 # error
 def invalid_primitive_schema_mismatch_template(field_name: str, data_type: str, data):
-    return (
-        "Datatype of field '{}' does not match the datatype '{}'. Data '{}'. "
-        "All dates/times/datetimes/durations must be ISO 8601 compliant.".format(
-            field_name, data_type, data,
-        )
+    return "Datatype of field '{}' does not match the datatype {}. Data sent by the device - {}. ".format(
```

Comment:
Is "field" the right language? Any other references I have seen refers to these elements as "property".

Reply:
I went with this because I thought there could be confusion with "device_properties". But its a nit, so feel free to change to property if we want to do that instead